edgelet/edgelet-core/src/lib.rs

```diff
@@ -96,3 +98,6 @@ impl UrlExt for Url {
 }
 
 pub const UNIX_SCHEME: &str = "unix";
+
+/// This is the name of the network created by the iotedged
+pub const DEFAULT_NETWORKID: &str = "azure-iot-edge";
```

Comment:
we have the same constant defined in edgelet-config\src\lib.rs; is it possible to reuse it? and maintain only 1 copy?

Reply:
Yep, I removed the one defined in that file and now we have just one copy in this file instead.